### PR TITLE
move from pages/integrations to models collection

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Notes for Reviewers**

This PR 
- removes all the pages/integrations markdown files
- initializes `models` collection, which has all the models' metadata in frontmatter variables.
> models collection can be accessed through `site.models` variable and filepath is : `docs/_models`
- updates site.pages with site.models for counting total number of integrations.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c87279be8061064e75cebb9c951d0f2512558046 at: https://6614f81d5c0801858ec408b8--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 22beef7e1cdd52b9d0d5cbf2a758a389cf47352c at: https://6614f9626bb17081c36cc5bc--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@vishalvivekm mere conflict.

File: docs/_config.yml [hunk 1]
```diff
@@ -99,6 +99,8 @@ collections:
     output: true
     permalink: reference/mesheryctl/:collection/:path
     name: mesheryctlcommands
+  models:
+    output: true  
   # integrations:
```
Comment (leecalcote) [inline docs/_config.yml:104]: @vishalvivekm why are we not uncommenting `integrations` and using that collection?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/_config.yml (hunk 1 above)
Comment (vishalvivekm) [reply docs/_config.yml]: @leecalcote `_integrations` is already a directory which has adapters metadata with versioning :
image

here's the link: 
https://github.com/meshery/meshery/tree/master/docs/_integrations

and yes `_adapters` is a collection dedicated to generate the pages.

Comment (leecalcote) [reply docs/_config.yml]: Not that integrations folder, but the integrations folder under extensibility, @vishalvivekm. You might need to cleanup where root of where `collections` are stored.

**Pull request closed without merging**